liquibase/changeLogs/capture/tables/aqToNwisParm/changeLog.yml

```diff
@@ -20,3 +20,18 @@ databaseChangeLog:
             path: aqToNwisParm.sql
             relativeToChangelogFile: true
         - rollback: drop table if exists ${AQTS_SCHEMA_NAME}.aq_to_nwis_parm;
+
+  - changeSet:
+      author: ayan
+      id: "alter.table.${AQTS_SCHEMA_NAME}.aq_to_nwis_parm.add.datum_unit"
+      preConditions:
+        - onFail: MARK_RAN
+        - onError: HALT
+        - not:
+          - columnExists:
+              schemaName: ${AQTS_SCHEMA_NAME}
+              tableName: aq_to_nwis_parm
+              columnName: datum
```

Comment:
The unit column is also being created. Do not know if liquidbase supports specifying more than one column. Maybe
multiple columnExists blocks.
If so, it can be added here. Else probably better to add the unit column as a separate changeLog.

Reply:
Good point - this does work locally, but I think it might just be easier to read if this is contained in separate changelogs too.